src/Commands/BatchProcess.php

```diff
@@ -186,13 +186,16 @@ protected function processBatch(BeamType $type): int
                                 'initialSupply' => $this->tokenCreatedCache[$key] ? null : $claim->quantity,
                                 'amount' => $this->tokenCreatedCache[$key] ? $claim->quantity : null,
                                 'cap' => [
-                                    'type' => $claim->quantity == 1 || $claim->collection?->force_single_mint ? TokenMintCapType::SINGLE_MINT->name : TokenMintCapType::INFINITE->name,
+                                    'type' => $claim->quantity == 1 || $claim->collection?->force_collapsing_supply ? TokenMintCapType::COLLAPSING_SUPPLY->name : TokenMintCapType::SUPPLY->name,
```

Comment:
The equivalent of `infinite` is just `null`, is the change for `supply` intended?

Reply:
Oh, i didn't know, thought it's collapsing supply. Will fix it